modules/ols-activating-token-quota-limits.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-activating-token-quota-limits_{context}"]
+= Activating token quota limits
+
+Activate token quota limits for the {ols-long} Service by defining key-value pairs in the `ConfigMap` resource. The {ols-long} pod mounts the `ConfigMap` resource as a volume, enabling access to the file stored within it. The `OLSConfig` Custom Resource (CR) references the `ConfigMap` resource to obtain the quota limit information.
+
+.Prerequisites
+
+* You have installed the the {ols-long} Operator.
+
+* You have configured a large language model provider (LLM).
+
+* A PostgresSQL database is configured and the {ols-long} Service can access the database.
+
+.Procedure
+
+. Open the {ols-long} `OLSconfig` CR file by running the following command:
++
+[source,terminal]
+----
+$ oc edit olsconfig cluster
+----
+
+. Modify the `data` property of the `spec.ols.quotaHandlersConfig` specification to include token quota limit information.
++
+.Example {ols-long} `OLSconfig` file
+[source,yaml]
+----
+apiVersion: v1
+kind: OLSconfig
+metadata:
+  name: quota-limit
+  namespace: openshift-lightspeed
+spec:
+  llm:
+  ols:
+    quota_handlers.conf: 
+      limiters:
+        - name: user_monthly_limits 
+          type: user_limiter
+          initial_quota: 100000 <1>
+          quota_increase: 10
+          period: 30 days
+        - name: cluster_monthly_limits 
+          type: cluster_limiter
+          quota_increase: 1000000 <2>
+          period: 30 days
+      scheduler:
+        period: 300 <3>
+
+    quotaHandlersConfig:
+      enableTokenHistory: true
+      limitersConfig: 
+        - initialQuota: 11111
+          name: default
+          period: 1d 
```

Comment:
@rh-tokeefe Just found out that this needs to be "1 day". 
1d doesn't allow us to save

Reply:
"1 d" would also be fine but the space is required.